docs/en/observability/logs-threshold-alert.asciidoc

```diff
@@ -4,10 +4,6 @@
 <titleabbrev>Log threshold</titleabbrev>
 ++++
 
-
```

Comment:
Note that we decided to remove the paths here and in other rules topics to be consistent across the rules documentation